posthog/management/commands/setup_test_environment.py

```diff
@@ -84,9 +85,28 @@ class MigrateSilentCommand(migrate.Command):
         def handle(self, *args, **kwargs):
             from django.db import connection
 
-            # :TRICKY: Create extension depended on by models.
+            from posthog.models.person import PersonOverride
+
+            create_function = f"""
+            SET check_function_bodies = false;
+            CREATE OR REPLACE FUNCTION is_override_person_not_used_as_old_person(override_person_id uuid, old_person_id uuid)
+            RETURNS BOOLEAN AS $$
+              SELECT NOT EXISTS (
+                SELECT 1
+                FROM "{PersonOverride._meta.db_table}"
+                WHERE override_person_id = $2
```

Comment:
Given that we're partitioning by team_id in the app does it add any value to include and index on `(team_id, override_person_id)` and update these lookups to include the team_id? I don't know exactly how that will help with looking speed.

At any rate we should update to filter by team_id here. It's obviously unlikely that we'd reuse a uuid but could happen accidentally I guess?

Reply:
> Not sure about making it multi-column though

Yeah my knowledge here is limited but I would imagine the composite would improve the lookup performance here and almost reduce the complexity to `log(number overrides in the team)` rather than `log(total number overrides)` but my postgresql/b-tree knowledge isn't great.

Anyway, it's not like we can't add this down the line.

Reply:
Makes sense, I would add two indexes `(team_id, override_person_id)` and `(team_id, old_person_id)`. We can do it later though, as we test performance of this table we can keep this in the back of our minds.

Reply:
We have an implicit index via the unique constraint on `(team_id, old_person_id)` so yeah just the `(team_id, override_person_id)`. Yup laterz.